### PR TITLE
Bug 1875397: Update wording in quick start panel

### PR DESCRIPTION
**Pull request by rottencandy**
**Fixes**: 
https://issues.redhat.com/browse/ODC-4642

**Analysis / Root cause**: 
- Text in quick start panel should show "quick start" instead of "tour".
- Text showing number of tasks should be singular when there is only one task.

**Solution Description**: 
- Replace "tour" with "quick start".
- Pluralize text showing number of tasks.

**Screen shots / Gifs for design review**: 
tmp0
-
tmp1

**Unit test coverage report**: 
Unchanged.

**Browser conformance**: 

- [ ] Chrome
- [x] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/console-app/src/components/quick-starts/controller/QuickStartIntroduction.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { SyncMarkdownView } from '@console/internal/components/markdown-view';
 import { QuickStartTask, QuickStartTaskStatus } from '../utils/quick-start-types';
 import TaskHeader from './QuickStartTaskHeader';
+import { pluralize } from '@patternfly/react-core';
```
Comment (abhinandan13jan) [inline frontend/packages/console-app/src/components/quick-starts/controller/QuickStartIntroduction.tsx:5]: not a big thing, but should this import be ordered at top.

Comment (rottencandy) [reply frontend/packages/console-app/src/components/quick-starts/controller/QuickStartIntroduction.tsx]: Updated.

**Review by rohitkrai03 — APPROVED** [review]
/approve

**Review by sahil143 — APPROVED** [review]
/lgtm

**Pull request merged**